specs/README.md

```diff
@@ -0,0 +1,18 @@
+# Component Specs
+
+This is the living space for the specs that lay the foundation on which our components are built. Each component should have a corresponding spec here that outlines at least, but is not limited to:
+
+- Component description
+- Variant considerations
+- Recommended API
+- Recommended DOM structure
+- Recommended slots
+- Behavioral information
+  - Component states
+  - Keyboard, cursor and touch interaction
+- Accessibility concerns
+  - Of the component developer
+  - Of the user
+- Component tokens
+
+## NOTE: The specs outlined here are currently a work in progress that shouldn't be taken as the source of truth until further announcement.
```

Comment:
Maybe put this at the top? Also a nitpicky thing, but we generally shouldn't be using headings for formatting. (though it's a bit unfortunate that markdown limits other ways to make things stand out)